installation/includes/01_default_config.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+RPI_BOOT_CONFIG_FILE=/boot/config.txt
```

Comment:
I would put this somewhere else. it's gotta be a constant but it shouldn't be changed by the normal user.. the other options in this file are meant to be changed.. thoughts?

Reply:
I'll have a look later tonight